freshmaker/pyxis.py

```diff
@@ -273,15 +273,16 @@ def get_images_by_digest(self, digest):
                           'filter': q_filter}
         return self._pagination('images', request_params)
 
-    def get_images_by_nvr(self, nvr):
+    def get_images_by_nvr(self, nvr, include_fields=[]):
```

Comment:
Mutable default arguments is a command python pitfall: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

And I'd suggest just use `include` instead of `include_fields` since Pyxis is using `include`.

Reply:
Yeah. A good point for the pitfall. I was considering to use a string, but I didn't choose it, because I need to tell the user to input a string with comma.

Reply:
Is it only for empty list default value, maybe include_fields=None?